neuro_extras/main.py

```diff
@@ -426,6 +471,10 @@ def init_aliases() -> None:
         "exec": "neuro-extras image copy",
         "args": "SOURCE DESTINATION",
     }
+    config["alias"]["storage-cp"] = {
```

Comment:
BTW what if alias `storage-cp` already exists? should we warn the user?

Reply:
That's a good question. I would prefer not to overwrite user's aliases, so perhaps we should fail here. Let's discuss this today after platform standup